ui/raidboss/emulator/ui/EmulatedPartyInfo.js

```diff
@@ -286,19 +347,54 @@ export default class EmulatedPartyInfo extends EventBus {
         false);
   }
 
-  _wrapCollapse(label, $obj, onclick) {
+  _wrapCollapse(params = {
+    time: undefined,
+    name: undefined,
+    icon: undefined,
+    text: undefined,
+    classes: undefined,
+    $obj,
```

Comment:
```suggestion
    $obj: undefined,
```

I am a little surprised that eslint `'object-shorthand': ['error', 'consistent'],` did not complain about this, but I think it should be consistent.

Reply:
That parameter is required which is why I didn't default it to undefined. I'm not sure how I would otherwise indicate as such without being able to use typescript.